### DISCUSSION TITLE
Hw 3 part 1

### DISCUSSION THREAD

**Discussion by ElisDiaz46** [cooperrc/computational-mechanics#171] [Q&A]
Hello, 

I am struggling to plot  Plot the polyfit lines for velocity and position . Here is my data 

 
![image](https://user-images.githubusercontent.com/116235922/211909035-6d271474-e6a7-449b-ae99-4b57bcd63053.png)

**Comment by cooperrc** [comment]
`np.polyfit` returns polynomial coefficients, so 

```python
>>> np.polyfit(t, x, 1)
[37.7, 1.8]
```

is telling you the best-fit line is

$x(t) = 37.7 t + 1.8~m$

where

$x(t) = v_0 t + x_0$
